src/Pages/EventManager/EventManager.js

```diff
@@ -48,7 +49,15 @@ export default function EventManager(props) {
 
   async function handleSubmit(event) {
     if (modalState === eventModalState.SUBMIT) {
-      await createNewEvent(event, props.user.token);
+      const res = await createNewEvent(event, props.user.token);
+      if(res.error === false) {
+        const addEventRes = await addEventToCalendar(event, props.user.token);
+        if(addEventRes.error === true) {
+          alert('Cannot add event to Google Calendar due to Event conflict!');
```

Comment:
after this pr lets use react to alert the user other than `alert`

Reply:
should these alerts be removed or kept in?